src/main/java/com/esri/samples/na/find_service_areas_for_multiple_facilities/FindServiceAreasForMultipleFacilitiesSample.java

```diff
@@ -0,0 +1,230 @@
+import com.esri.arcgisruntime.symbology.SimpleRenderer;
+import com.esri.arcgisruntime.tasks.networkanalysis.ServiceAreaParameters;
+import com.esri.arcgisruntime.tasks.networkanalysis.ServiceAreaPolygon;
+import com.esri.arcgisruntime.tasks.networkanalysis.ServiceAreaPolygonDetail;
+import com.esri.arcgisruntime.tasks.networkanalysis.ServiceAreaResult;
+import com.esri.arcgisruntime.tasks.networkanalysis.ServiceAreaTask;
+
+public class FindServiceAreasForMultipleFacilitiesSample extends Application {
+
+  private MapView mapView;
+
+  @Override
+  public void start(Stage stage) {
+
+    // create stack pane and application scene
+    StackPane stackPane = new StackPane();
+    Scene scene = new Scene(stackPane);
+    scene.getStylesheets().add(getClass().getResource("/css/style.css").toExternalForm());
+
+    // set title, size, and add scene to stage
+    stage.setTitle("Find Service Area for Multiple Facilities Sample");
+    stage.setWidth(800);
+    stage.setHeight(700);
+    stage.setScene(scene);
+    stage.show();
+
+    // create button
+    Button findServiceAreasButton = new Button("Find Service Areas");
+    findServiceAreasButton.setMaxWidth(150);
+    findServiceAreasButton.setDisable(true);
+
+    // create a progress indicator
+    ProgressIndicator progressIndicator = new ProgressIndicator();
+    progressIndicator.setVisible(false);
+
+    // create an ArcGISMap with a streets basemap
+    ArcGISMap map = new ArcGISMap(Basemap.createLightGrayCanvas());
+
+    // set the ArcGISMap to be displayed in the map view
+    mapView = new MapView();
+    mapView.setMap(map);
+
+    // create a graphics overlay for displaying service areas
+    GraphicsOverlay serviceAreasGraphicsOverlay = new GraphicsOverlay();
+
+    // add the graphics overlay to the map view
+    mapView.getGraphicsOverlays().add(serviceAreasGraphicsOverlay);
+
+    // create fill symbols for rendering the result
+    ArrayList<SimpleFillSymbol> fillSymbols = new ArrayList<>();
+    fillSymbols.add(new SimpleFillSymbol(SimpleFillSymbol.Style.SOLID, 0x66FFA500, null));
+    fillSymbols.add(new SimpleFillSymbol(SimpleFillSymbol.Style.SOLID, 0x66FF0000, null));
+
+    // create a feature table of facilities using a FeatureServer
+    ArcGISFeatureTable facilitiesTable = new ServiceFeatureTable("https://services2.arcgis.com/ZQgQTuoyBrtmoGdP/ArcGIS/rest/services/San_Diego_Facilities/FeatureServer/0");
+    // create a feature layer from the table
+    FeatureLayer facilitiesFeatureLayer = new FeatureLayer(facilitiesTable);
+
+    // create a symbol used to display the facilities
+    PictureMarkerSymbol facilitySymbol = new PictureMarkerSymbol("http://static.arcgis.com/images/Symbols/SafetyHealth/Hospital.png");
+    facilitySymbol.setHeight(25);
+    facilitySymbol.setWidth(25);
+    // set the renderer of the facilities feature layer to use the facilities symbol
+    facilitiesFeatureLayer.setRenderer(new SimpleRenderer(facilitySymbol));
```

Comment:
The above lines are unecessary for the basics of the sample, I'm not sure whether you should say that in a comment like:
"override the default symbol used to display facilities"